templates/index.html

```diff
@@ -0,0 +1,112 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="../css/style.css" />
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    <p>Hello World</p>
+    <!--
+
+    Hello Camper!
+
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+
+    -->
+    <div class=nav>
+
+        <ul>
+          <li class="listitems">
+            <a href="#">About</a>
+          </li>
+          <li class="listitems">
+            <a href="#">work</a>
+          </li>
+          <li class="listitems">
+            <a href="#">contact</a>
+          </li>
+        </ul>
```

Comment:
- [ ] I think it will be better to wrap your list of navigation lists at nav that's better for SEO(search engine optimization) 👌, so I suggest you wrap your list at semantic element called nav. for more information about semantic elements [Click here](https://www.w3schools.com/html/html5_semantic_elements.asp) 📖